src/blockchain/any.rs

```diff
@@ -52,7 +52,7 @@
 //!
 //! # #[cfg(feature = "esplora")]
 //! # {
-//! let esplora_blockchain = EsploraBlockchain::new("...");
+//! let esplora_blockchain = EsploraBlockchain::new("...", None);
```

Comment:
Might be a good idea to just put `"https://blockstream.info/testnet/api"` as first argument so people can copy-paste from the documentation.